include/envoy/stats/stats.h

```diff
@@ -94,5 +94,18 @@ class Gauge : public virtual Metric {
 
 typedef std::shared_ptr<Gauge> GaugeSharedPtr;
 
+/**
+ * A Boolean.
+ */
+class BoolIndicator : public virtual Metric {
+public:
+  virtual ~BoolIndicator() {}
+
+  virtual void set(bool value) PURE;
+  virtual uint64_t value() const PURE;
```

Comment:
why do we set with a bool, but pull out the value with a uint64_t? Can we just have value() return a bool too?

Reply:
Sounds good, will do.